app/scripts/services/tile-proxy.js

```diff
@@ -2,7 +2,7 @@ import { range } from 'd3-array';
 import slugid from 'slugid';
 
 import {
-  workerFetchTiles, // eslint-disable-line import/named
+  // workerFetchTiles,
```

Comment:
```suggestion
```

This could just be deleted: See #737

Reply:
I leave it in as commented out code because other commented-out functions rely on it. The code needs proper cleaning in another PR, just removing this line makes things even more confusing.